docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/amazon-quicksight.md

```diff
@@ -0,0 +1,118 @@
+
+
+
+
+## User management and permissions
+
+
+When working with analytics, users can have the following roles:
+* Author: can view, create, and edit assets.
+* Reader: can view assets.
+
+<!--
+To give a Back Office user access to analytics, you need to assign one of these roles to them. For instructions, see [Create users]() and [Edit users]().
+
+-->
+
+The analytics roles are derived from Amazon QuickSight and are separate from the [Back Office user roles](/docs/pbc/all/user-management/{{page.version}}/base-shop/manage-in-the-back-office/best-practices-manage-users-and-their-permissions-with-roles-and-groups.html).
+
+
+## Analyses and dashboards
+
+In Amazon QuickSight, you are going to work with analyses and dashboards.
+
+An analysis is a customizable visual representation of data. Using the provided tools, you can manipulate your data to extract and present the most important information.
+
+Each analysis can contain multiple data visualizations, which can be rearranged and customized.
+
+
+![QuickSight-analyses](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/amazon-quicksight.md/qs-analysis.png)
+
+
+A dashboard is a representation of data prepared in an analysis. It serves as a published version of data analysis, which can be shared with other users.
+
+![QuickSight-dashboard](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/amazon-quicksight.md/qs-dashboard.png)
+
+
+Both analyses and dashboards are shareable, you can share a complete dashboard or collaborate on an analysis.
+
+![QuickSight-sharing](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/amazon-quicksight.md/qs-sharing.png)
+
+You can share dashboards with authors and readers, and analyses – only with authors.
+
+### Default dashboards
+
+For those new to Amazon QuickSight analytics, while you're learning how to create your own dashboards, you can start using the default dashboards right away.
+
+The feature is shipped with the following dashboards.
+
+#### Sales dashboard
+
+* Track sales performance over time
+* Gain insights into orders and return rate
+* Monitor discounts, product categories, brands
+* Monitor the usage of payment and shipping methods
+
+#### Products dashboard
+
+* Track performance of products
+* Monitor availability and returns
+* Analyze top-selling items, category distribution, and trends in shopping lists and customer carts
+
+
+#### Customers dashboard
+
+Track the following customer metrics:
+* Order number and value
+* Registration trends
+* Locations
+* Demographics
+
+
+#### Marketplace dashboard
+* Track marketplace performance with insights into merchant status and commission
+* Get an overview of merchant orders, products, and offers
+
+
```

Comment:
New section to be added here:
"Synchronizing users"
TBD

Reply:
Please provide more details on how the new user management system works, including the following details:
1. What users do I have after setting up QS?
2. What happens to the user that activates QS?
3. How do I add new users?
4. How do I manage user roles?

And any other details because I have 0 info on the new user management

Reply:
Maybe here we can add if you need more users, please contact CSM as we did in the part of describing how to get Analytics in general.